packages/terra-clinical-onset-picker/tests/jest/__snapshots__/OnsetPicker.test.jsx.snap

```diff
@@ -51,7 +51,7 @@ exports[`correctly applies the theme context className 1`] = `
           aria-disabled="false"
           aria-expanded="false"
           aria-haspopup="true"
-          aria-label="Precision, Select a date precision"
+          aria-label="Precision, Precision"
```

Comment:
Is this aria-label correct?
CC: @scottwilmarth

Reply:
Yes, one `Precision is from label`, the other is from the placer holder.

Reply:
@sdadn I think this is a perfect example of how we must choose better words when creating props. The aria-label in, this case, is not actual aria-label, but the prop where the first word displays as the placeholder text and the second displays as the visible field label. @vinaybhargavar please correct me if I'm wrong.

Regarding the concern for passivity, these fields will fail accessibility requirements without the faceup label. There is no way to make this a passive change and be accessible. Without the visible label, assistive technology users may not know how to activate the controls - specifically, a speech input device user requires a visible label to know how to call out the controls to activate them.

Reply:
@scottwilmarth - I don't think that `aria-label` is a prop. We don't set the aria-label here. The clinical onset picker internally utilizes the `SelectField` component from `terra-form-select` which sets the aria-label.

@sdadn - The aria-label remains the same as before, just that the label was changed from Select a date precision to just precision for this field.

Reply:
@sdadn, @scottwilmarth - To illustrate that the first string is the value of the combo box, and the second string is the label of the combo box, see this line in the snapshot - https://github.com/cerner/terra-clinical/pull/894/files#diff-058379a6a77388b3d64c973d6b9c88743d1472471b2f8ae25d5ad3ed0a3b2b6fR450. First it says `Unknown Selected`, then the label `precision`.

Reply:
@sycombs - I see that Saad is OOO until Thursday afternoon. Can you take a look at this and let me know your thoughts? TIA.